basis/ui/tools/listener/listener-docs.factor

```diff
@@ -97,4 +97,8 @@ TIP: "Scroll the listener from the keyboard by pressing " { $command listener-ga
 
 TIP: "Press " { $command tool "common" refresh-all } " or run " { $link refresh-all } " to reload changed source files from disk. " ;
 
+os windows? [
+    TIP: "Use C+Break to interrupt tight loops in your code started in the Listener, such as" { $code "[ t ] [ ] while" } "Caution: this may crash the Factor runtime if the code uses cooperative multitasking or asynchronous I/O." ;
+] when
+
```

Comment:
You could just say, "On Windows, ..." or (currently supported on Windows only..." etc